### PR TITLE
reward in US dollar, Rate API, different reward interest

### PR DESCRIPTION
**Pull request by longwinterday**

### CODE REVIEW

File: rewards/tasks.py [hunk 1]
```diff
@@ -118,7 +137,39 @@ async def check_waiting_airdrops():
...
+        rates = await config.api.rates
+        for i_currency in rates:
+            decimals = DECIMALS.get(i_currency, 0)
+            rate, _ = await Rate.get_or_create(currency=i_currency, decimals=decimals)
+            rate.usd_rate = rates.get(i_currency).get("USD")
+            await rate.save()
+    except Exception as err:
+        logger.warning("Cant get rates from API cause {err}".format(err=err))
+    rate = await Rate.get(currency=currency)
+    return int(10**rate.decimals / rate.usd_rate)
```
Comment (ephdtrg) [inline rewards/tasks.py:176]:
```suggestion
    return int(10 ** rate.decimals / rate.usd_rate)
```

File: rewards/tasks.py [hunk 2]
```diff
@@ -6,8 +6,10 @@
 from tortoise.transactions import atomic
```
Comment (ephdtrg) [inline rewards/tasks.py:6]:
```suggestion
from tortoise.transactions import atomic
from tortoise import timezone
```

File: rewards/tasks.py [hunk 3]
```diff
@@ -43,16 +42,34 @@ async def ping_nodes() -> None:
...
-        peer, _ = await Peer.get_or_create(enode=enode)
+        peer, _ = await Peer.get_or_create(
+            enode=enode,
+            defaults={
+                "reward_interest": round(config.default_usd_reward_amount / 100, 18)
+            },
+        )
 
         is_online = enode in active_enodes
+        timestamp = datetime.now() - timedelta(days=1)
```
Comment (ephdtrg) [inline rewards/tasks.py]:
```suggestion
        timestamp = timezone.now() - timedelta(days=1)
```

File: rewards/tasks.py (hunk 1 above)
Comment (longwinterday) [reply rewards/tasks.py:176]: А тута блэк по пепу стайлит https://peps.python.org/pep-0008/#other-recommendations

**Review by ephdtrg — APPROVED** [review]

**Pull request merged**